simulariumio/file_converter.py

```diff
@@ -59,25 +59,6 @@ def _update_trajectory_info_v1_to_v2(data: Dict[str, Any]) -> Dict[str, Any]:
             "magnitude": time_units.magnitude,
             "name": time_units.name,
         }
-        # default camera transform
-        data["trajectoryInfo"]["cameraDefault"] = {
-            "position": {
-                "x": DEFAULT_CAMERA_SETTINGS.CAMERA_POSITION[0],
-                "y": DEFAULT_CAMERA_SETTINGS.CAMERA_POSITION[1],
-                "z": DEFAULT_CAMERA_SETTINGS.CAMERA_POSITION[2],
-            },
-            "lookAtPosition": {
-                "x": DEFAULT_CAMERA_SETTINGS.LOOK_AT_POSITION[0],
-                "y": DEFAULT_CAMERA_SETTINGS.LOOK_AT_POSITION[1],
-                "z": DEFAULT_CAMERA_SETTINGS.LOOK_AT_POSITION[2],
-            },
-            "upVector": {
-                "x": DEFAULT_CAMERA_SETTINGS.UP_VECTOR[0],
-                "y": DEFAULT_CAMERA_SETTINGS.UP_VECTOR[1],
-                "z": DEFAULT_CAMERA_SETTINGS.UP_VECTOR[2],
-            },
-            "fovDegrees": DEFAULT_CAMERA_SETTINGS.FOV_DEGREES,
-        }
```

Comment:
I ran into an issue while converting v2 files that didn't have a cameraDefault, so instead of only adding it during version updating (here), I changed it to create a default one in `CameraData.from_buffer_data()`. This matches the way I'm handling this for `ModelMetaData` in this PR, and makes it so the defaults get populated anytime it's missing in the old file, regardless of the version it started as.